crates/extensions/aws_mapper_ext/src/converter.rs

```diff
@@ -21,16 +21,18 @@ pub struct AwsConverter {
     pub(crate) clock: Box<dyn Clock>,
     pub(crate) size_threshold: SizeThreshold,
     pub mqtt_schema: MqttSchema,
+    pub topic_root: String,
```

Comment:
The same argument is given twice: first as an `MqttSchema`, second as `String`. Keep only the first one.